Makefile

```diff
@@ -0,0 +1,11 @@
+BINARY := keystone-auth-hook
+TARGET := build/$(BINARY)
+
+$(TARGET):
+	glide install
```

Comment:
Better add '--strip-vendor' flag to remove nested vendor directories after install.